### PR TITLE
creating help and contributing pages

### PR DESCRIPTION
**Pull request by abaird**
moves over all of the content I could find from watir-webdriver and the
watir.com site

### CODE REVIEW

File: help.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: help
```
Comment (thekiiingbob) [inline help.md]: "help" should probably be capitalized. On the page it displays as lower case.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Contributing
+---
+
+## I think I found a bug:
+Submit an [issue](https://github.com/watir/watir-webdriver/issues) or [pull request](https://github.com/watir/watir-webdriver/pulls) to the [watir-webdriver project](https://github.com/watir/watir-webdriver).
+
+## I want to contribute to the documentation:
+The public webpage is maintaned as a GitHub page written in Jekyll. To contribute, send a PR at [our webpage](https://watir.github.io).
```
Comment (thekiiingbob) [inline contributing.md]: Should the link at the end of this line go to the github repo instead of back to the homepage?

**Comment by abaird** [comment]
Bob - thanks for the suggestions, I have fixed them above

**Comment by thekiiingbob** [comment]
:+1:  Looks good to :ship:

**Pull request merged**